cmd/wio/toolchain/npm/resolve/local.go

```diff
@@ -55,14 +58,19 @@ func tryFindConfig(name string, ver string, dir string) (*types.PkgConfig, error
             return nil, errors.Stringf("config %s has wrong name", paths[i])
         }
         if tryConfig.Version() != ver {
-            if i != 0 {
-                return nil, errors.Stringf("config %s has wrong version", paths[i])
-            } else {
-                // version-less path
-                continue
-            }
+            return nil, errors.Stringf("config %s has wrong version", paths[i])
         }
         config = tryConfig
+
+        if i == 0 || i == 1 {
+            // package is found in vendor directory
+            node.Vendor = true
+        } else {
```

Comment:
can you change this function to check a specific path? that way this logic can sit outside the function

i.e. `tryFindConfig(name, ver, path) (*types.PkgConfig, error)` where `path` is a specific path like `<project>/vendor/<name>_<version>` and it tries to grab `wio.yml`